build.gradle.kts

```diff
@@ -26,6 +26,7 @@ dependencies {
     api("com.github.stephenc.jcip:jcip-annotations:1.0-1")
     api(webdriver("selenium-api"))
 
+    implementation("com.atlassian.performance:selenium-js:[1.0.0,2.0.0)")
```

Comment:
What's the upside of including a unused lib?

Reply:
I want to make it available to people who depend on jira-software-actions to promote best practices.

Reply:
Then do `jira-software-actions` fail to use the best practices?

Reply:
:-) With the amount of polling in jsa, it does not make sense to switch to the native conditions.